osu.Game.Tournament/Screens/StablePathSelectScreen.cs

```diff
@@ -0,0 +1,177 @@
+                                new Drawable[]
+                                {
+                                    new FillFlowContainer
+                                    {
+                                        Anchor = Anchor.Centre,
+                                        Origin = Anchor.Centre,
+                                        Direction = FillDirection.Horizontal,
+                                        Spacing = new Vector2(20),
+                                        Children = new Drawable[]
+                                        {
+                                            new TriangleButton
+                                            {
+                                                Anchor = Anchor.Centre,
+                                                Origin = Anchor.Centre,
+                                                Width = 300,
+                                                Text = "Select stable path",
+                                                Action = () => ChangePath(storage)
+                                            },
+                                            new TriangleButton
+                                            {
+                                                Anchor = Anchor.Centre,
+                                                Origin = Anchor.Centre,
+                                                Width = 300,
+                                                Text = "Auto detect",
+                                                Action = AutoDetect
+                                            },
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                },
+                new BackButton
+                {
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    State = { Value = Visibility.Visible },
+                    Action = () => sceneManager?.SetScreen(typeof(SetupScreen))
+                }
+            });
+        }
+
+        protected virtual void ChangePath(Storage storage)
+        {
+            var target = directorySelector.CurrentDirectory.Value.FullName;
+            stableInfo.StablePath.Value = target;
+            Logger.Log($"Changing Stable CE location to {target}");
+
+            if (!FileBasedIPC.CheckExists(target))
+            {
+                overlay = new DialogOverlay();
+                overlay.Push(new IPCErrorDialog("This is an invalid IPC Directory", "Select a directory that contains an osu! stable cutting edge installation and make sure it has an empty ipc.txt file in it."));
+                AddInternal(overlay);
+                Logger.Log("Folder is not an osu! stable CE directory");
+                return;
+                // Return an error in the picker that the directory does not contain ipc.txt
```

Comment:
This comment reads as if there is something missing here, which I don't think is the case (there's no error being returned). I'd just remove it.